### DISCUSSION TITLE
Add a numeric prefix to the migration file name to facilitate ascending sorting within the folder.

### DISCUSSION THREAD

**Discussion by Korolev-Oleg** [sqlalchemy/alembic#1241] [Ideas]
Currently, because a random hash is used at the beginning of the migration name, they are also displayed in a random sequence like the screenshot below.
![image](https://github.com/sqlalchemy/alembic/assets/18235953/465c90c9-f9a4-4d05-a529-ec0121b11e8c)
I suggest adding an index `00001` before the migration name in order to maintain the sequence.

**Comment by Korolev-Oleg** [comment]
You can make changes to the hash generation algorithm and modify the first characters so that they form a sequence of numbers. `00001aa4131f`

**Comment by CaselIT** [comment]
The name is already customizable by the user. Look for `file_template` in https://alembic.sqlalchemy.org/en/latest/tutorial.html#editing-the-ini-file
